task2/src/models/Movie.js

```diff
@@ -1,20 +1,51 @@
+import PropTypes from 'prop-types'
+import { VOTE_AVERAGE_MIN, VOTE_AVERAGE_MAX } from '@utils/constants'
+import { range } from '@utils/customs'
+
 class Movie {
     constructor(
         title,
+        tagline,
+        voteAverage,
+        voteCount,
         releaseDate,
+        posterPath,
+        overview,
+        budget,
+        revenue,
         genres,
-        poster,
-        rating,
-        duration,
-        description) {
+        runtime,
+        id) {
         this.title = title;
+        this.tagline = tagline;
+        this.voteAverage = voteAverage;
+        this.voteCount = voteCount;
         this.releaseDate = releaseDate;
+        this.posterPath = posterPath;
+        this.overview = overview;
+        this.budget = budget;
+        this.revenue = revenue;
         this.genres = genres;
-        this.poster = poster;
-        this.rating = rating;
-        this.duration = duration;
-        this.description = description;
+        this.runtime = runtime;
+        if (id) {
+            this.id = id;
+        } // JS cannot overload constructors :(
```

Comment:
what do u mean?

1. u can add default arguments for constructor
2. u can add such thing here, if u don't want `id` in any case: 
```
id && (this.id = id)
```
or u can use `??` nulish operator instead `&&`